checkbox.js

```diff
@@ -18,13 +18,13 @@ class CheckBox extends Component {
 
         this.state = {
             internalChecked: false
-        }
+        };
 
         this.onChange = this.onChange.bind(this);
     }
 
     onChange() {
-        if (this.props.checked && this.props.onChange) {
+        if (typeof this.props.checked == 'boolean' && this.props.onChange) {
```

Comment:
If you set it via checked to 'true' it would work. If you set it to `false` it would break into the next statement, and who knows?